google/cloud/security/scanner/scanner.py

```diff
@@ -195,12 +190,8 @@ def _get_project_policies(timestamp):
         The project policies.
     """
     project_policies = {}
-    try:
-        dao = project_dao.ProjectDao()
-        project_policies = dao.get_project_policies('projects', timestamp)
-    except MySQLError as err:
```

Comment:
I see the tweak now.  I guess i need to pull from master.  Regardless, this makes sense now.  👍